about.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="styles.css" />
+        <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@700&family=Roboto:ital,wght@0,100;1,100&display=swap"
+            rel="stylesheet"
+        />
+    </head>
+    <body class="container">
+        <div class="first-header">
+            <div class="face-twitter">
+                <img src="images/Rectangle.svg" alt="" />
+                <img src="images/twitter.svg" alt="" />
+            </div>
+            <div class="first-nav">
+                <ul>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </div>
+
+        <header class="header">
+            <div class="menu-logo">
+                <img
+                    src="images/mobile_menu 1.svg"
+                    alt=""
+                    onclick="popMenu()"
+                />
+            </div>
+
+            <div class="pop-menu" id="menu-pop">
+                <i
+                    class="fa fa-times"
+                    aria-hidden="true"
+                    id="menu-close"
+                    onclick="closePop()"
+                ></i>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>Sponsor</li>
+                </ul>
+            </div>
```

Comment:
- [OPTIONAL] There's no link back to the home page from about page on your mobile version. I tried to navigate back to the home page after clicking on the hamburger, but no menu item to handle that. I marked it optional because, it escaped the sight of the previous reviewer. Kindly treat it as matter of importance.

### Your App in Action
[your-app-about-mobile.webm](https://github.com/Developer-Kingz/Microverse-capstone/assets/26597432/ef74d471-e676-4271-adfa-c389d377ad79)